vscode-trace-extension/src/utils/backend-tsp-client-provider.ts

```diff
@@ -9,11 +9,11 @@ import { TspClientProvider } from 'vscode-trace-common/lib/client/tsp-client-pro
  *  for the Trace Extension's VSCode Backend
  */
 
-let _root = getUriRootFromUserSettings();
-let _path = getApiPathFromUserSettings();
-let _url = _root + _path;
-
-const _provider = new TspClientProvider(_url, undefined);
+let _extUri: vscode.Uri;
```

Comment:
It seems this could be a local variable, defined inside `updateTspClientUrl()`?

Reply:
Went ahead and changed this as you suggested.

After I made this change, it seems the linting errors went away?  I ran eslint `yarn lint` and `yarn format:check` and no errors came up.

Reply:
That seems almost too good to be true - I will confirm here :)